modules/aerospike/aerospike_test.go

```diff
@@ -0,0 +1,64 @@
+package aerospike_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	as "github.com/testcontainers/testcontainers-go/modules/aerospike"
+)
+
+const (
+	aerospikeImage = "aerospike/aerospike-server:latest"
+)
+
+// TestAerospike tests the Aerospike container functionality
+// It includes tests for starting the container with a valid image,
+// applying container customizations, and handling context cancellation.
+// It also includes a test for an invalid image to ensure proper error handling.
+// The tests use the testcontainers-go library to manage container lifecycle
+func TestAeroSpike(t *testing.T) {
+	t.Run("fails_with_invalid_image", func(t *testing.T) {
```

Comment:
suggestion: more concise subtests names, using `-` instead of `_`
```suggestion
	t.Run("invalid-image-fails", func(t *testing.T) {
```